www/webkit2-gtk/files/patch-WTF-wtf-Randomdevice.diff

```diff
@@ -0,0 +1,115 @@
+diff --git Source/WTF/wtf/RandomDevice.cpp Source/WTF/wtf/RandomDevice.cpp
+index 61d6057..90fa30c 100644
+--- Source/WTF/wtf/RandomDevice.cpp
++++ Source/WTF/wtf/RandomDevice.cpp
+@@ -30,24 +30,12 @@
+ #include <stdint.h>
+ #include <stdlib.h>
+ 
+-#if !OS(DARWIN) && OS(UNIX)
+ #include <errno.h>
+ #include <fcntl.h>
+ #include <unistd.h>
+-#endif
+-
+-#if OS(WINDOWS)
```

Comment:
@kencu: I steal this patch from your repo :) The question is: is it possible to get an patch without touching WINDOWS bits? (I can't test this as this patch is used for older systems only)

Reply:
Yeah, maybe. My brain was getting scrambled trying to sort out the `#ifdefs` so I just stripped the parts I didn't want right out of the file. I wish there was some kind of editor that could hide and show the file parts depending on the value of the `#ifdefs`. Maybe there is on :>